Sources/_CFXMLInterface/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ target_include_directories(_CFXMLInterface
         ../CoreFoundation/include
     PRIVATE
         ../CoreFoundation/internalInclude
-        /usr/include/libxml2/)
+        ${LIBXML2_INCLUDE_DIR})
```

Comment:
Where is this defined?

Reply:
it's defined by [`find_package(LibXml2)`](https://cmake.org/cmake/help/latest/module/FindLibXml2.html)